client/app/templates/templates.service.js

```diff
@@ -94,4 +95,20 @@ export function TemplatesServiceFactory(CollectionsApi) {
 
     return CollectionsApi.post(collection, template.id, {}, editObj);
   }
+
+  function deleteTemplates(templates) {
+    const templateIds = [];
+    templates.forEach(function (template) {
```

Comment:
This is copied exactly how it was done in the catalog state service.  I can change it up but if we change it here, it needs to be changed there as well.